dds/DCPS/Service_Participant.cpp

```diff
@@ -2065,6 +2065,45 @@ NetworkConfigMonitor_rch Service_Participant::network_config_monitor()
   return network_config_monitor_;
 }
 
+#if defined(OPENDDS_NETWORK_CONFIG_UPDATER)
```

Comment:
In network_config_monitor (above in file), there should be else if defined (ACE_HAS_IOS) or whatever and the network_config_monitor_ should be initialized with a NetworkConfigUpdater.